src/js/components/SearchBar.jsx

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react';
 import TextField from 'material-ui/TextField';
 
 import { FUZZY_SEARCH_URL } from '../settings';
+import searchIcon from '../../img/search-icon.png';
```

Comment:
It might be useful to just install a font into the project, or use a cdn. Font Awesome is what we use for Mobile.

MaterialUI also comes with built in icon support, with built in [SVGs](http://www.material-ui.com/#/components/svg-icon). Although their icon doesn't look like the one you got (your one looks more like font awesome).